### ISSUE TITLE
installation via npm fails

### ISSUE THREAD

**Issue by kremalicious** [zeusdeux/isInViewport#28]
With node v6.3.1 & npm v3.10.3 the installation of this package fails:

```shell
npm i is-in-viewport
```

results in:

```shell
npm ERR! is-in-viewport@2.4.1 postinstall: `grunt`
npm ERR! spawn ENOENT
npm ERR! Failed at the is-in-viewport@2.4.1 postinstall script 'grunt'.
```

**Comment by goodslav** [comment]
I can confirm the same

**Comment by zeusdeux** [comment]
This is due to npm/npm#10366. 
Fixed in 948503a87de236e2692901f31193bcb56ba95192. Thanks!

**Comment by kremalicious** [comment]
nice, thank you for the fix, Sir!

![200](https://cloud.githubusercontent.com/assets/90316/17555092/70041576-5f0e-11e6-91f4-6a5e077ed105.gif)
